src/components/Agreements/ViewAgreement/Sections/Organizations.js

```diff
@@ -24,32 +25,64 @@ export default class Organizations extends React.Component {
     }).isRequired,
     id: PropTypes.string,
     onToggle: PropTypes.func,
+    organizations: PropTypes.object,
     open: PropTypes.bool,
   };
 
-  renderOrgList = (orgs) => {
+  renderOrgList = () => {
+    const { organizations } = this.props;
     return (
       <React.Fragment>
-        {orgs.map(o => (
-          o.org ?
-            <Layout data-test-organizations-org className="marginTopHalf" key={`${o.org.id}-${o.role.value}`}>
-              {o.org.orgsUuid ?
-                <Link to={`/organizations/view/${o.org.orgsUuid}`}>{o.org.name}</Link> :
-                o.org.name
-              }
-              {o.role && `, ${o.role.label}`}
-            </Layout> : null
-        ))}
+        {organizations && organizations.map(o => {
+          const interfaces = get(o.org, ['orgsUuid_object', 'interfaces'], []);
+          return (
+            o.org ?
+              <Card
+                data-test-organizations-org
+                key={`${o.org.id}-${o.role.value}`}
+                header={
+                  <span className="my-card-header">
+                    {
+                      o.org.orgsUuid ?
+                        <Link to={`/organizations/view/${o.org.orgsUuid}`}>
+                          {o.org.name}
+                          .
+                          {o.role.label}
+                        </Link> : o.org.name
+                    }
+                  </span>
+                }
+              >
+                {interfaces && interfaces.length ?
+                  <MultiColumnList
+                    contentData={interfaces}
+                    visibleColumns={['name', 'uri', 'username', 'password', 'type']}
```

Comment:
Mockup looks like there's no URI column; the Name is a link to the URI.

Reply:
Also, should there be a `Note` column?

Reply:
The description here says otherwise https://issues.folio.org/browse/ERM-165. Went with that.

Reply:
Can you double check with Owen what it's supposed to be? Leave a note on JIRA maybe since he's OOO I believe. We can merge without confirming, just want to make sure it's logged and he's aware for when he's doing acceptance testing.

Reply:
Added a comment to the JIRA story.